.github/workflows/merge-gate.yml

```diff
@@ -0,0 +1,30 @@
+name: Merge-Gate
+
+on:
+  workflow_call:
+    secrets:
+      ALTERNATIVE_GITHUB_TOKEN:
+        required: false
+
+jobs:
+
+  fast-checks:
+    name: Fast
+    uses: ./.github/workflows/checks.yml
+
+  #slow-checks:
```

Comment:
I think it's a good idea to leave this code here and comment it out. At some point, we may move the integration tests to a GitHub large runner. I would add a comment about this here.